### ISSUE TITLE
WFS sample is not working. 

### ISSUE THREAD

**Issue by pauldendulk** [Mapsui/Mapsui#1464]
De xml parser returns:
System.Exception: 'A service exception occured: Unknown namespace [p_bz-cadastre]'

As a consequence I can not test the NTS port for WFS. I do not want to dig into this right now so created this issue to keep track of it.

**Comment by janusw** [comment]
Yeah, I also noticed this recently. Looks like the layers have changed in the WFS service that is used there. I'm looking into it ...
